manifests/init.pp

```diff
@@ -377,6 +382,8 @@
   $puppet_wsgi_processes     = $pulp::params::puppet_wsgi_processes,
   $migrate_db_timeout        = $pulp::params::migrate_db_timeout,
   $show_conf_diff            = $pulp::params::show_conf_diff,
+  $enable_profiling          = $pulp::params::enable_profiling,
+  $profiling_directory       = $pulp::params::profiling_directory,
```

Comment:
Would be good to have some validations on these, validate_bool validate_absolute_path respectively.